package.json

```diff
@@ -22,7 +22,9 @@
     "@angular/platform-browser": "12.2.0",
     "@angular/platform-browser-dynamic": "12.2.0",
     "@angular/router": "12.2.0",
+    "bootstrap": "^5.1.3",
     "eslint": "8.1.0",
+    "ngx-owl-carousel-o": "^6.0.0",
```

Comment:
Please remove unused carousel dependency

Reply:
Removed it